File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/xrts/XRT0101U00GrdMcodeInfo.java
package nta.med.data.model.ihis.xrts;

public class XRT0101U00GrdMcodeInfo {
	private String codeType ;
	private String codeTypeName ;
	public XRT0101U00GrdMcodeInfo(String codeType, String codeTypeName) {
		super();
		this.codeType = codeType;
		this.codeTypeName = codeTypeName;
	}
	public String getCodeType() {
		return codeType;
	}
	public void setCodeType(String codeType) {
		this.codeType = codeType;
	}
	public String getCodeTypeName() {
		return codeTypeName;
	}
	public void setCodeTypeName(String codeTypeName) {
		this.codeTypeName = codeTypeName;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Xrts/XRT0101U00GrdMcodeInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Xrts
{
    [Serializable]
    public class XRT0101U00GrdMcodeInfo
    {
        private String _codeType;
        private String _codeTypeName;

        public String CodeType
        {
            get { return this._codeType; }
            set { this._codeType = value; }
        }

        public String CodeTypeName
        {
            get { return this._codeTypeName; }
            set { this._codeTypeName = value; }
        }

        public XRT0101U00GrdMcodeInfo() { }

        public XRT0101U00GrdMcodeInfo(String codeType, String codeTypeName)
        {
            this._codeType = codeType;
            this._codeTypeName = codeTypeName;
        }

    }
}